src/Lykke.AlgoStore.Api/Controllers/AlgoController.cs

```diff
@@ -165,13 +165,13 @@ public async Task<IActionResult> AddToPublic([FromBody] PublicAlgoDataModel mode
         {
             var clientId = User.GetClientId();
             var data = Mapper.Map<PublicAlgoData>(model);
+            data.ClientId = clientId;
```

Comment:
You can remove the clientId variable and call the extension method directly here